view/frontend/web/js/product/button.js

```diff
@@ -29,21 +30,20 @@ define([
                 }
             });
 
-            inputElements = _.every(inputElements);
-
-            $.each(radioAndCheckbox,function (i,el) {
-                _.contains(el,true) ? radioAndCheckbox[i] = true : radioAndCheckbox[i] = false;
+            var inputElementsSet = _.every(inputElements);
```

Comment:
personally I don't know if this really seems like a boolean variable name, it seems like a Set data structure instead, something starting like areInputElementsSet would be clearer I think